test/verify/check-apps

```diff
@@ -65,14 +67,14 @@ class TestApps(packagelib.PackageCase):
     <pkgname>{p}</pkgname>
   </component>
 """
-        self.machine.execute("mkdir -p /usr/share/app-info/xmls")
+        self.machine.execute(f"mkdir -p /usr/share/{subdir}")
         self.createPackage("appstream-data-test", "1.0", "1", content={
-            "/usr/share/app-info/xmls/test.xml": f"""
+            f"/usr/share/{subdir}/test.xml": f"""
 <components origin="test">
 {body}
 </components>
             """,
-            "/usr/share/app-info/icons/test/64x64/test.png": {"path": "/var/tmp/test.png"}})
+            f"/usr/share/{subdir.split('/')[0]}/icons/test/64x64/test.png": {"path": "/var/tmp/test.png"}})
```

Comment:
this is a bit brittle, but I'll allow it.

Reply:
Yeah, it's a bit meh, but I didn't want to make these two parameters, as they are really not independent. At least the test will clearly fail if this is wrong (which was what led me to [fix this](https://github.com/cockpit-project/cockpit/compare/58238f9803af7b6e93160a96a98fb667131f706b..ef7d4c6e0d96e8878ba9cb1ac23a896ef6398b86) in the first place).